Doc/library/stdtypes.rst

```diff
@@ -804,7 +804,7 @@ number, :class:`float`, or :class:`complex`::
            hash_value = -2
        return hash_value
 
-.. _typebool:
+.. _bltin-boolean-values:
```

Comment:
This might break people who've already adapted to the new link anchor. You can have them both, like this:

```suggestion
.. _bltin-boolean-values:
.. _typebool
```

That would also mean that you wouldn't have to update `functions.rst`, you'd only have to update this file

Reply:
Are people already using the new anchor only 2 days after release?

But okay, I can try it your way. Thanks!

Reply:
> Are people already using the new anchor only 2 days after release?

No idea, but it seems possible! And this way means a smaller diff ;)